deeplearning4j/dl4j-integration-tests/src/test/java/org/deeplearning4j/integration/IntegrationTestBaselineGenerator.java

```diff
@@ -56,8 +59,8 @@
 @Slf4j
 public class IntegrationTestBaselineGenerator {
 
-    public static final File OUTPUT_DIR_DL4J = new File("../../dl4j-test-resources/src/main/resources/dl4j-integration-tests").getAbsoluteFile();
-    public static final File OUTPUT_DIR_SAMEDIFF = new File("../../dl4j-test-resources/src/main/resources/samediff-integration-tests").getAbsoluteFile();
+    public static final File OUTPUT_DIR_DL4J = new File("/home/atuzhykov/SkyMind/dl4j-test-resources/src/main/resources/dl4j-integration-tests").getAbsoluteFile();
+    public static final File OUTPUT_DIR_SAMEDIFF = new File("/home/atuzhykov/SkyMind/dl4j-test-resources/src/main/resources/samediff-integration-tests").getAbsoluteFile();
```

Comment:
Let's make sure we revert this change before committing.
Note that it _should_ work fine for you with `../../dl4j-test-resources` if you have both dl4j-test-resources and deeplearning4j repos cloned in the same parent directory